package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "relay-subscriptions",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
not sure what your publish pipeline is, but doesn't `npm version` usually take care of this for you?